### PR TITLE
create payment success page

### LINKED ISSUE #47: Create PaymentSucceful Page
#18 

Completed Payment

### PR DESCRIPTION
**Pull request by abdulrahman-2020**
create payment success page

- [x] create payment success page with Html
- [x] create payment success page with css

### CODE REVIEW

File: PaymentSucceful/PaymentSucceful.html [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```
Comment (EmanHass) [inline PaymentSucceful/PaymentSucceful.html]: change the title, set it Payment

File: PaymentSucceful/PaymentSucceful.html [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    <link rel="stylesheet" href="PaymentSucceful.css">
```
Comment (EmanHass) [inline PaymentSucceful/PaymentSucceful.html]: set href='./PaymentSucceful.css'

**Review by EmanHass — COMMENTED** [review]
Be careful, you created a branch in another branch

**Review by MMansour87 — APPROVED** [review]
Well Done

**Review by EmanHass — APPROVED** [review]

**Review by EmanHass — APPROVED** [review]

**Review by EmanHass — APPROVED** [review]

**Review by mo7amedehab97 — APPROVED** [review]

**Review by lina-jamal — APPROVED** [review]

**Pull request merged**